opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -19,60 +19,53 @@
  * <p>Spans are created by the {@link Tracer#buildSpan} interface.
  */
 public interface Span extends AutoCloseable {
+    /**
+     * Retrieve the associated SpanContext.
+     *
+     * @return the SpanContext that encapsulates Span state that should propagate across process boundaries.
+     */
+    SpanContext getContext();
```

Comment:
ps the history of getX setX is JavaBeans. this isn't a java bean, so
doesn't need to behold to that. A lot of the api has more to do with
builders than beans
http://www.informit.com/articles/article.aspx?p=1216151&seqNum=2